.build/.k.rev

```diff
@@ -1 +1,5 @@
+<<<<<<< HEAD
 be708fe8ea35de3b4d54d21780548fc12b2faf20
+=======
+964eee2490bdb177b37f7069be14dae5fd6f91bd
+>>>>>>> 0217104... fix ambiguous lemma
```

Comment:
this is not correct. you have to properly resolve the cherry-pick conflict.

Reply:
indeed, thank you